From df67ca9882b4d42f905b38cc9888060c7b26fbde Mon Sep 17 00:00:00 2001
From: Jacob Martin <jjm_223@hotmail.com>
Date: Mon, 17 Jul 2017 17:28:41 -0500
Subject: [PATCH] Fix invisible blocks

Fix taken from the description here: https://www.youtube.com/watch?v=lu2Dgpf2yRM
Imgur link to fix: http://i.imgur.com/AdvUzmE.png

diff --git a/src/main/java/net/minecraft/server/PlayerInteractManager.java b/src/main/java/net/minecraft/server/PlayerInteractManager.java
index a1689c06..38b35123 100644
--- a/src/main/java/net/minecraft/server/PlayerInteractManager.java
+++ b/src/main/java/net/minecraft/server/PlayerInteractManager.java
@@ -195,7 +195,7 @@ public class PlayerInteractManager {
                 this.f = blockposition;
                 int i = (int) (f * 10.0F);
 
-                this.world.c(this.player.getId(), blockposition, i);
+                this.world.notify(blockposition, iblockdata, iblockdata, 0);
                 this.k = i;
             }
 
-- 
2.17.0

